run-server

```diff
@@ -1,3 +1 @@
-#!/bin/sh
-
-cd Avocado && yarn start && cd ..
+python -m webbrowser https://www.youtube.com/watch?v=DLzxrzFCyOs
```

Comment:
Why does run server play a Youtube video?

Reply:
Did you run it?